.mergify.yml

```diff
@@ -1,9 +1,13 @@
 pull_request_rules:
   - name: automatic merge on CI success and review
     conditions:
-      - status-success=continuous-integration/circleci/pr
+      - status-success=cla/google
+      - status-success=ci/circleci
```

Comment:
I think it needs to exactly match the name? It is (hopefully) fixed this in master so if you rebase it may work

Reply:
rebased.